docs/Fields.md

```diff
@@ -283,6 +283,34 @@ See [Intl.Numberformat documentation](https://developer.mozilla.org/en-US/docs/W
 
 **Tip**: If you need more formatting options than what `Intl.Numberformat` can provide, build your own field component leveraging a third-party library like [numeral.js](http://numeraljs.com/).
 
+## `<PhoneField>`
+
+`<PhoneField>` displays a phone number as a `<a href="tel:" />` link.
+
+If the number is from the locale it'll be formatted like a national number, if not international formatting will be set.
```

Comment:
You could add that the component accepts local and international phone number as long as the locale is the one set in the props.